samples/bot-calling-meeting/csharp/Source/CallingBotSample/Bots/MessageBot.cs

```diff
@@ -256,17 +222,128 @@ private async Task SendResponse(ITurnContext<IMessageActivity> turnContext, stri
             }
         }
 
-        private async Task SendMeetingActionsCard(ITurnContext turnContext, CancellationToken cancellationToken)
+        private async Task<TResult> MakeGraphCallThatMightNotBeFound<TResult>(Func<Task> function, string? callId, Func<string, Task<TResult>> errorHandler)
```

Comment:
Please document-- the method name reads oddly :)

Reply:
hope this makes things clearer